src/components/ButtonBuilder.js

```diff
@@ -0,0 +1,31 @@
+function ButtonBuilder(props) {
+  const { id, quantity, handleEvents } = props;
+  function handleClick(e, id) {
```

Comment:
> Don't define the helper functions inside a functional component because with component re-rendering there'll be a new instance creation of it. Still we need to discuss about this.